docs/设计文档.md

```
@@ -0,0 +1,93 @@
+# TinyPro 接口文档
+
+本文档旨在捋清TinyPro后端设计架构。
+
+## 表设计
+
+![](./imgs/TinyPro.png)
+
+## 专用术语
+
+- 公开接口: 一个不需要Token的接口, 一般是登录接口.
+- 非公开接口: 一个需要Token的接口, 当Token过期时必须返回401响应码. 错误报文格式请参考[异常格式](#异常格式)
```

Markdownlint (0.35.0)

Link fragments should be valid

(MD051, link-fragments)